soroban-spec/fixtures/ts/src/convert.ts

```diff
@@ -0,0 +1,109 @@
+import { Address, xdr } from 'soroban-client';
+import { Buffer } from "buffer";
+import { bufToBigint } from 'bigint-conversion';
+
+export function scvalToBigInt(scval: xdr.ScVal | undefined): BigInt {
+    switch (scval?.switch()) {
+        case undefined: {
+            return BigInt(0);
+        }
+        case xdr.ScValType.scvU64(): {
+            const { high, low } = scval.u64();
+            return bufToBigint(new Uint32Array([high, low]));
+        }
+        case xdr.ScValType.scvI64(): {
+            const { high, low } = scval.i64();
+            return bufToBigint(new Int32Array([high, low]));
```

Comment:
this isn't quite right: low is uint32, high is int32

Reply:
Hmmm well I don't have an obvious solution to this.  I think the whole convert piece will be changed in the next release.

Reply:
Yeah def, just wanted to call this out for when stellar/js-stellar-base#620 gets in